lib/platform/github/index.js

```diff
@@ -210,6 +210,21 @@ async function initRepo({
       );
       throw new Error('archived');
     }
+    try {
+      const renovateConfig = JSON.parse(
+        Buffer.from(
+          (await get(
+            `repos/${config.repository}/contents/${defaultConfigFile}`
+          )).body.content,
+          'base64'
+        ).toString()
+      );
+      if (!renovateConfig.enabled) {
+        throw new Error();
+      }
+    } catch (err) {
+      throw new Error('config-disabled');
```

Comment:
```suggestion
      // Do nothing - continue as if we never tried
```

Reply:
@rarkins should not we consider JSON.parse() errors? There might be a possibility that config file checked in by the user cannot be parsed because it's not in a proper format.